sklearn/tree/_classes.py

```diff
@@ -112,16 +112,26 @@ def __init__(self,
         self.ccp_alpha = ccp_alpha
 
     def get_depth(self):
-        """Returns the depth of the decision tree.
+        """Return the depth of the decision tree.
 
         The depth of a tree is the maximum distance between the root
         and any leaf.
+
+        Returns
+        -------
+        self.tree_.max_depth : int or None
```

Comment:
That attribute is defined in `sklearn/tree/_tree.pyx` and I think it only be an int? 

(If the tree is not fitted an error would be raised).

Reply:
After instantiating, but before fitting, it's `None` by default?
e.g.
```>>> from sklearn import tree
>>> X = [[0, 0], [1, 1]]
>>> Y = [0, 1]
>>> clf = tree.DecisionTreeClassifier()
>>> type(clf.max_depth)
NoneType```

Reply:
yes, but we are documenting the `get_depth` method here, and that one never return None I think. When not fitted it raises a `NotFittedError`,
```py
>>> from sklearn.tree import DecisionTreeClassifier
>>> tree = DecisionTreeClassifier()
>>> tree.get_depth()
Traceback (most recent call last):
[...]
sklearn.exceptions.NotFittedError
```